src/tests/ftest/dfuse/read.py

```diff
@@ -0,0 +1,96 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import time
+
+from dfuse_test_base import DfuseTestBase
+from run_utils import run_remote
+
+
+class DFuseReadTest(DfuseTestBase):
+    """Base ReadTest test class.
+    :avocado: recursive
+    """
+
+    def test_dfuse_pre_read(self):
+        """
+        Test Description:
+            Ensure that pre-read feature is working.
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=dfuse
+        :avocado: tags=DFuseReadTest,test_dfuse_pre_read
+        """
+
+        pool = self.get_pool(connect=False)
+        container = self.get_container(pool)
+
+        self.load_dfuse(self.hostlist_clients, None)
+
+        self.dfuse.disable_wb_cache.value = True
+
+        self.dfuse.env["D_LOG_MASK"] = "INFO,DFUSE=DEBUG"
+        self.dfuse.env["DD_MASK"] = "ALL"
+        self.dfuse.env["DD_SUBSYS"] = "ALL"
```

Comment:
Similarly, we can set these in the config as
```
dfuse:
  client:
    env_vars:
    - D_LOG_MASK=INFO,DFUSE=DEBUG
    - DD_MASK=ALL
    - DD_SUBSYS=ALL
```

Reply:
We probably don't need these anymore now the test is complete/working.  I'll remove but yes, I would have put these in the config if I knew the correct syntax.